clearnode/docs/API.md

```diff
@@ -398,6 +411,99 @@ Retrieves the user's tag, which can be used for transfer operations. The tag is
 }
 ```
 
+### Get Session Keys
+
+Retrieves all active (non-expired) session keys for the authenticated user. Session keys are delegated keys that can perform operations on behalf of the user's wallet with spending caps and expiration times.
+
+This endpoint returns only session keys, not custody signers. Each session key includes its spending allowances and current usage tracking.
+
+**Request:**
+
+```json
+{
+  "req": [1, "get_session_keys", {}, 1619123456789],
+  "sig": ["0x9876fedcba..."]
+}
+```
+
+**Response:**
+
+```json
+{
+  "res": [1, "get_session_keys", {
+    "session_keys": [
+      {
+        "id": 1,
+        "session_key": "0xabcdef1234567890...",
+        "application_name": "Chess Game",
+        "allowance": [
+          {
+            "asset": "usdc",
+            "amount": "100.0"
+          },
+          {
+            "asset": "eth",
+            "amount": "0.5"
+          }
+        ],
+        "used_allowance": [
+          {
+            "asset": "usdc",
+            "amount": "45.0"
+          },
+          {
+            "asset": "eth",
+            "amount": "0.0"
+          }
+        ],
```

Comment:
Wouldn't it be more useful to provide both "allowance" and "used_allowance" in the same structure?
This way the referenced response will look like:
```json
"allowance": [
  {
    "asset": "usdc",
    "allowance": "100.0",
    "used_allowance": "45.0",
  },
  {
    "asset": "eth",
    "allowance": "0.5",
    "used_allowance": "0"
  }
],
```